### ISSUE TITLE
Compiler error: too few arguments to function ‘panel_applet_setup_menu_from_file’

### ISSUE THREAD

**Issue by fmmarzoa** [rcaelers/workrave#108]
**Describe the bug**
I get a compiler error when building the last version from the sources. I have tried with the last 1.10 branch and also checking out v1_10_20 tag with the same result.

The first error to shown is:

```
v4/WorkraveApplet.c:441:3: error: too few arguments to function ‘panel_applet_setup_menu_from_file’
   panel_applet_setup_menu_from_file(PANEL_APPLET(applet), ui_path, applet->priv->action_group);

```

Others follow that one, but seems related to it.

See at the attached make.out file for the complete output.

**To Reproduce**
1. I attach my configure.ac file that should be put in place of the original configure.ac. The differences are basically that I enable gnome3 configuration and lib_panelapplet4.

2. run ./autogen.sh
3. run ./configure.sh
4. run make

**Expected behavior**
Successful compilation

** Linux (please complete the following information in case you encountered the bug on Linux):**
 - Distribution: Ubuntu 
 - Distribution Version: 18.04
 - Desktop Environment: Gnome
 - Workrave Version 1.10/1.10.20

[report.tar.gz](https://github.com/rcaelers/workrave/files/2274297/report.tar.gz)

**Comment by rcaelers** [comment]
On Ubuntu 18.04 you need libpanel-applet v5. The versioning Workrave is using is confusing.

But so is the gnome-panel versioning. The successor of libpanel-applet-4 is apparently called libpanel-applet3

History versions:
Ubuntu 14.04: libpanel-applet-4-0   3.8.0 
Ubuntu 16.04 : libpanel-applet0         3.18.2
Ubuntu 18.04: libpanel-applet3         3.26.0

Workrave should automatically select the correct versions is all the libraries have been installed.

**Comment by fmmarzoa** [comment]
Thanks Rob! 

Yes, the versioning of libpanel-applet it's a hell of confusing. The one installed in my system is libpanel-applet3. It seems there are no official packages for libpanel-applet5 for Ubuntu 18.04, so I'll take it from somewhere else. I hope it works at run time, I really don't know if it's compatible with the Gnome version installed.

I'm closing this report, then.

**Comment by rcaelers** [comment]
libpanel-applet3 is the correct version, Workrave incorrectly calls it v5... There is no libpanel-applet5 it seems. My mistake. So no need to get it from somewhere else. Just use the one from Ubuntu

**Comment by fmmarzoa** [comment]
So I must enable libpanel-applet5 in the configure.ca file?

**Comment by fmmarzoa** [comment]
Well, I did just that, compiles and installs, but it shows an error on Gnome Tweak/Extensions so I cannot enable the it to show the tray icon.

I found out this error in gnome-shell log:

Extension "workrave@workrave.org" had error: Error: Requiring Workrave, version none: Typelib file for namespace 'Workrave' (any version) not found

Workrave is there, nonetheless, and it opens without problems if I invoke it from the command line. Any idea on how to solve this?

TIA!
